python/paddle/fluid/tests/book/high-level-api/fit_a_line/test_fit_a_line.py

```diff
@@ -72,11 +71,7 @@ def event_handler(event):
             '''
             if float(test_metrics[0]) < 20.0:
                 if save_dirname is not None:
-                    # NOT clear yet
-                    # fluid.io.save_inference_model(save_dirname, ['x'], [y_predict])
-                    # trainer.save_params(save_dirname)
-                    # https://github.com/PaddlePaddle/Paddle/pull/10445
-                    trainer.save_inference_model(save_dirname)
+                    trainer.save_params(save_dirname)
```

Comment:
I noticed we switched from `trainer.save_inference_model(save_dirname)` back to `trainer.save_params(save_dirname)`. What is the diff between them? We discussed this [here](https://github.com/PaddlePaddle/Paddle/pull/10445#discussion_r187501346). To me [save_inference_model](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/trainer.py#L223) looks more reasonable.

Reply:
https://github.com/PaddlePaddle/Paddle/issues/10648
@wangkuiyi Think that we only need to save parameters and do not need to save inference model, inference model should be provided by a Python function.